docs: heater-shaker instruction manual

jwwojak opened the pull request:
# Overview

This migrates the [Heater-Shaker manual](https://insights.opentrons.com/hubfs/Products/Modules/Heater-Shaker%20Manual.pdf) to MkDocs.

RTC-781

## Test Plan and Hands on Testing

Review for the usual.

## Changelog

<!--
List changes introduced by this PR considering future developers and the end user. Give careful thought and clear documentation to breaking changes.
-->

## Review requests

**Image resize problem**
Having a lot of trouble with resizing the image `adapter-attachment.png` in the `adapters.md` section. This is in an ordered list / procedure list and normally gets indented to match the list. However, when indented the image always renders as full size.

I tried to left justify the image, and then it resizes properly, but the then the warning admonition that follows it breaks and renders as a code block.

Temporary fix:
- left justify the image and set a size
- move the admonition above/before the image so it can be indented with the list text and still work.
- not sure how to fix this. please take a look.

<img width="635" height="607" alt="adapter attach issue" src="https://github.com/user-attachments/assets/ca83787d-4295-44a4-a053-99c5b5814dad" />

**LED table alignment problem**
In `specifications.md` the amber dot and text don't align. However, this should be fixed on merge because in a previous merge for another module, I added `status-dot-container` CSS to handle this.

<img width="786" height="175" alt="alignment" src="https://github.com/user-attachments/assets/354cae49-02bb-4b5b-876d-70f4de3162a3" />


## Risk assessment

Low, docs only.

jwwojak committed:
Created new branch for HS manual

jwwojak committed:
Yaml file formatted and saving

jwwojak committed:
Early afternoon commit

Because, reasons.

jwwojak committed:
Maintenance and cleaning

jwwojak committed:
Warranty page

jwwojak committed:
Safety and compliance commit 1

jwwojak committed:
Compliance text 2

Tables and images

jwwojak committed:
Back-matter and compliance sections

jwwojak committed:
Finish product specifications section

jwwojak committed:
Thermal adapter start

jwwojak committed:
Adapters section - no images

Need to add images.

jwwojak committed:
Preinstall start

jwwojak committed:
Mid-morning save

jwwojak committed:
Lunch commit

jwwojak committed:
End of day commit

jwwojak committed:
Mid-afternoon save

jwwojak committed:
Lunch commit

jwwojak committed:
Merge branch 'edge' into heater-shaker-instruction-manual

jwwojak committed:
Fixing merge conflicts

Strange leading spaces in

- docs/shared/modules/index.md

and

- docs/mkdocs.yml

jwwojak committed:
Fix adapter attachment image

jwwojak committed:
Fixing adapter attachment image and warning

jwwojak commented:
Adapter image re-sizing problem resolved w/ help from @ecormany (thank you). However, I still have to put the warning admonition before the image. When indenting it to match the ordered list, which you can do but not in this case, MkDocs renders it as a code block. Putting the admonition before the list solves the problem and allows you to indent the warning.

Also, the spacing in the LED table is resolved by CSS I added to `docs/shared/instruction-manuals.css` in a previous push (maybe for the Thermocycler). This:

```
 /* Aligns status dots and text if needed*/
.status-dot-container {
  display: flex;
  align-items: center;
  gap: 5px;
}
``` 

This is ready for review.

jwwojak committed:
Add unicode and test CSS

jwwojak committed:
Merge branch 'edge' into heater-shaker-instruction-manual

ecormany commented on docs/heater-shaker/docs/index.md:
```diff
@@ -0,0 +1,21 @@
+
+<div style="text-align: center;" markdown>
+
+![Opentrons Flex®](../images/opentrons-flex-logo.svg "opentrons-flex-logo.svg"){style="width: 60%"}
```
Needs to be just the Opentrons logo. H-S works on Flex and OT-2.

jwwojak replied:
Really? This means we have to change the covers for the other modules that work with OT-2 as well. I don't think it is worth it. Let's not go down this path. One cover style to rule them all.

ecormany commented on docs/heater-shaker/docs/compliance.md:
```diff
@@ -0,0 +1,135 @@
+      <td>40-60%, non-condensing</td>
+      <td>80%</td>
+      <td>10-85%, non-condensing (below 30 °C)</td>
+    </tr>
```
keeping these ranges consistent. i like "to" when we've got a negative number in the mix. something like "–10–+60" is 😵‍💫 
```suggestion
    <tr>
      <td>Ambient Temperatures</td>
      <td>20 to 24 °C</td>
      <td>20 to 25 °C</td>
      <td>–10 to +60 °C</td>
    </tr>
    <tr>
      <td>Relative Humidity</td>
      <td>40 to 60%, non-condensing</td>
      <td>80%</td>
      <td>10 to 85%, non-condensing (below 30 °C)</td>
    </tr>
```

ecormany commented on docs/heater-shaker/docs/compliance.md:
```diff
@@ -0,0 +1,135 @@
+    <tr>
+      <td>Relative Humidity</td>
+      <td>40-60%, non-condensing</td>
+      <td>80%</td>
```
hang on. this is what's in the PDF but it can't be right. there has to be a range here, not a single point!

jwwojak replied:
You're referring to the 80% relative humidity?

To make sure this wasn't a transcription error, I went back to the [draft doc](https://docs.google.com/document/d/16hdHGX4IbtA5zNNs4j5BREIOW8sWMLv7EAnWbYmOUc4/edit?usp=sharing). 80% is what we have. Maybe this means "up to" 80%. 

I can check w/ hardware. However, my recommendation is to not let this hold us up too much.

ecormany replied:
Looked in the GDoc history and an older version had `80% maximum`! That seems right, let's go with it.

ecormany commented on docs/heater-shaker/docs/compliance.md:
```diff
@@ -0,0 +1,135 @@
+### Safety
+
+- IEC/EN 61010-1 Safety­ Requirements­ for­ Electrical­ Equipment for­ Measurement,­ Control­, and Laboratory Use
+- IEC ­ 61010-2-010­ Requirement ­for­ Heating
```
remove extra space
```suggestion
- IEC 61010-2-010­ Requirement ­for­ Heating
```

ecormany commented on docs/heater-shaker/docs/compliance.md:
```diff
@@ -0,0 +1,135 @@
+
+### CISPR 11 Class A
+
+**Caution:** This equipment is not intended for use in residential environments and may not provide adequate protection to radio reception in such environments.
```
Should add the standard WEEE section below this.

ecormany replied:
Oh, I see it's at the end. That's fine. Or could move it here.

jwwojak replied:
Seems like "compliance" is a better place for the WEEE notification. I will move it here. We need to get consistent with this. When WEEE stuff first appeared, this was your suggestion to put it at the end.

ecormany replied:
I think putting it at the end was because we were inserting it in print publications and didn't want to have to re-flow and re-layout all the pages.

ecormany commented on docs/heater-shaker/docs/index.md:
```diff
@@ -0,0 +1,21 @@
+---
+title: "Heater-Shaker Module"
```
I've been using the full publication name on the home page of each publication.

Then on the other pages, I think it's better to shorten to `Heater-Shaker` instead of `Heater-Shaker Module`.

jwwojak replied:
Will fix.

ecormany commented on docs/heater-shaker/docs/specifications.md:
```diff
@@ -0,0 +1,164 @@
+    </tr>
+    <tr>
+      <td>Maximum</td>
+      <td>125-130 W</td>
```
en dashes 😇 
```suggestion
          <li>Shaking: 4–11 W</li>
          <li>Heating: 10–30 W</li>
          <li>Heating and Shaking: 10–40 W</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>Maximum</td>
      <td>125–130 W</td>
```

ecormany commented on docs/heater-shaker/docs/specifications.md:
```diff
@@ -0,0 +1,164 @@
+      </td>
+    </tr>
+    <tr>
+      <td><span class="status-dot-container"> <!-- testing split color dot -->
+        <span class="status-dot yellow-red-split"></span> Amber/Red</span></td>
+      <td>
+        <ul>
+          <li>Solid amber indicates an error.</li>
+          <li>Pulsing red and amber (alternating) indicates an error <em>and</em> the module is hot to the touch (>49 °C)</li>
```
I like the split dot, nice. But I think we should have an extra row here: one with an all-amber dot, and one with the split dot.

jwwojak replied:
Will have a go at it.

ecormany commented on docs/heater-shaker/docs/adapters.md:
```diff
@@ -0,0 +1,49 @@
+[opentronsTough]: https://labware.opentrons.com/nest_96_wellplate_100ul_pcr_full_skirt?category=wellPlate#/?loadName=opentrons_96_wellplate_200ul_pcr_full_skirt
+[corning384]: https://labware.opentrons.com/nest_96_wellplate_100ul_pcr_full_skirt?category=wellPlate#/?loadName=corning_384_wellplate_112ul_flat
+
+The Universal Flat Bottom Plate Adapter works with most flat- bottom ANSI/SLAS automation compliant labware. For more information, see the JSON Labware Schema section in the Labware chapter of the Flex Instruction Manual.
```
can link now!!
```suggestion
The Universal Flat Bottom Plate Adapter works with most flat- bottom ANSI/SLAS automation compliant labware. For more information, see the [JSON Labware Schema section](../flex/labware/definitions.md#json-labware-schema) in the Labware chapter of the Flex Instruction Manual.
```

jwwojak replied:
Thanks for the link!

ecormany commented on docs/heater-shaker/mkdocs.yml:
```diff
@@ -0,0 +1,53 @@
+  - Safety Information and Regulatory Compliance: compliance.md
+  - Product Specifications: specifications.md
+  - Thermal Adapters: adapters.md
+  - Installing the Heater-Shaker:
```
I'm torn on how to name pages like this. 
- In context, just `Installation` is much cleaner in the left nav.
- In search, the nav title is used, and we could wind up with a bunch of results all saying "Installation".

jwwojak replied:
Bullet 2 is a reason not to do this. I recommend keeping the "Installing the <_insert module name here_> title format.

ecormany commented on docs/heater-shaker/docs/preinstall.md:
```diff
@@ -0,0 +1,75 @@
+
+## Flex Caddies
+
+When used with a Flex robot, the Heater-Shaker fits into a caddy that occupies space below the deck. The caddy places your labware closer to the deck surface and allows for below-deck cable routing. See the Caddies section in Chapter 4 of the Flex Instruction Manual for more information.
```
Also linkable!
```suggestion
When used with a Flex robot, the Heater-Shaker fits into a caddy that occupies space below the deck. The caddy places your labware closer to the deck surface and allows for below-deck cable routing. See the [Caddies section](../flex/modules/caddy.md) of the Flex Instruction Manual for more information.
```

jwwojak replied:
Thanks for the link!

ecormany commented on docs/heater-shaker/docs/images/module-status.png:
Wow this image has been through hell and back. I think I took the original screenshot, so I'll see if I can dig it up.

ecormany reviewed (commented):
Yep, that's the H-S manual! Just a few suggestions.

ecormany commented on docs/mkdocs.yml:
```diff
@@ -4,12 +4,13 @@ docs_dir: ./shared
 
 nav:
-  - Opentrons Flex: '!include flex/mkdocs.yml'
+  - Opentrons Flex: '!include flex-manual/mkdocs.yml'
```
Be sure to not change this from the original (via rebase, manual change, whatever). The correct path now is `flex/mkdocs.yml`.

jwwojak replied:
I didn't change this. Will set it to `flex/mkdocs.yml` as shown above.

<img width="819" height="423" alt="Screenshot 2025-08-13 at 9 49 19 AM" src="https://github.com/user-attachments/assets/4f24fb3f-8b5c-4354-b715-53d8e24a90d5" />

jwwojak committed:
Applying suggestions from code review

Will make others after pulling these changes.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak reviewed (commented):
Committing review changes and making others separately.

jwwojak committed:
Committing review changes

Additional changes:

- Fix `include` path to flex manual
- Make humidity spec "80% maximum"
- In index.md, use full title in front matter
- In specifications.md, add separate rows for amber and amber/red
- Move WEEE policy text to compliance.md
- Fix path to flex manual in mkdocs.yml

jwwojak committed:
Remove logo image from front cover

ecormany reviewed (approved):
Looks good!
